### ISSUE TITLE
Is there a way to chande rb_color attribute from java code?

### ISSUE THREAD

**Issue by Shunajlov** [skyfishjy/android-ripple-background#31]
Is there a way to chande rb_color attribute not from xml?

**Comment by sinadalvand** [comment]
i have same issue , is there any way?

**Comment by rostopira** [comment]
Looks like this repo isn't maintained anymore
So I've wrote kotlin extension for this

```
val RippleBackground.paint: Paint? get() =
    try {
        val field = RippleBackground::class.java.getDeclaredField("paint")
        field.isAccessible = true
        field.get(this) as? Paint
    } catch (e: Exception) {
        null
    }
```
Use it like this `ripple.paint?.color = resources.getColor(R.color.very_red)`

And don't forget to add proguard rule, just to be safe
```
-keepclassmembers class com.skyfishjy.library.RippleBackground {
    private android.graphics.Paint paint;
}
```

**Comment by ManoR88** [comment]
Yes there is a complex way to do that you have to copy java class and change few codes....

![image](https://user-images.githubusercontent.com/43945872/61293625-c1afed80-a7f1-11e9-9cc1-a05b1b9dd4b0.png)
rb_color is green in xml

![image](https://user-images.githubusercontent.com/43945872/61293787-1489a500-a7f2-11e9-88e9-cc668f418cda.png)
rb_color is red in java class

End result
![image](https://user-images.githubusercontent.com/43945872/61293931-5fa3b800-a7f2-11e9-9016-45642e70e25d.png)

**Comment by ManoR88** [comment]
Yes there is a complex method.... Just copy RippleBackground class and few changes in code to change rb_color attribute from java code..
